src/autocomplete/autocomplete.js

```diff
@@ -31,6 +32,13 @@ export default ({
   const [isLoading, setIsLoading] = useState(false)
   const inputRef = useRef()
 
+  // call user-supplied onFeatures callback
+  if (typeof userOnFeatures === 'function') {
```

Comment:
the `if typeof` check should be moved _into_ the effect, react does not like conditionally applying effects like this.